doc/lua_api.txt

```diff
@@ -6947,6 +6950,9 @@ Used by `minetest.register_node`.
 
         on_rightclick = function(pos, node, clicker, itemstack, pointed_thing),
         -- default: nil
+        -- Called when clicker (a player object) "rightclicks"
```

Comment:
`on_punch` is also missing a description
it might also be better not to suggest that `clicker` must always be a player object